src/states/LevelOne.js

```diff
@@ -1,13 +1,46 @@
-var LevelOne = function () {};
+/* globals GUIBuilder */
+var LevelOne = function () {
+  this.gui = null;
+};
 
 LevelOne.prototype = {
 
   preload: function() {
+    game.stage.backgroundColor = '#444444';
+    game.load.script('guiBuilder', 'src/guiBuilder.js');
+  },
+
+  toggleView: function(btn) {
+    var group = btn.parent;
+    var groupPos = group.x === 0 ? -game.world.width : 0;
+    var btnRot = group.x === 0 ? -3 * Math.PI : 0;
 
+    game.add.tween(group).to({x: groupPos}, 1200, Phaser.Easing.Quadratic.In, true);
+    game.add.tween(btn).to({rotation: btnRot}, 1200, Phaser.Easing.Quadratic.In, true);
   },
```

Comment:
Sure! Tweening is just interpolation from one value to another. In this case I am interpolating the x coordinate of the group of buttons and the rotation of the button pressed (the arrow). Normally something like this would need to be done in the update function, but luckily Phaser provides us with a function to make things easier. 

So whats happening is that the tween function is that it is taking in an object that will have a property to be interpolated, the following function call is to specify which value to actually change and where it should go.
* As you can see the first argument of `to` is an object containing the value(s) to change and their new value
* The second argument is the amount of time in milliseconds for the interpolation to happen over
* The third argument is the type of easing function to use. An easing functions just refers to the change in velocity with respect to time of the value, in this case in moves with quadratic acceleration.
* The final argument is whether or not to autostart the tween

If you're interested, here's the [wikipedia article on tweening](https://en.wikipedia.org/wiki/Inbetweening).